<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>9</y>
         <width>311</width>
         <height>461</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="bt_ver">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string>Ver</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_write">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>130</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string>Escribir</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_out_pg">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string>Out</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_search_pg">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>390</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>119</x>
         <y>19</y>
         <width>721</width>
         <height>441</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QTextEdit" name="text_bbox">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>60</y>
           <width>441</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(239, 239, 239);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>160</y>
           <width>194</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(214, 214, 214);</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>220</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <item>
          <property name="text">
           <string>T1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T3</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="bt_enviar">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>400</y>
           <width>96</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(255, 89, 196, 255));</string>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>81</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>User</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>200</y>
           <width>81</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Turno</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>58</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
        <widget class="QLabel" name="st_enviar">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>270</y>
           <width>181</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QPushButton" name="bt_out">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>401</y>
           <width>81</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Out</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bt_ver_3">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>400</y>
           <width>111</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(255, 253, 121, 255));
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Ver</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="text_bbox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>341</width>
           <height>351</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(239, 239, 239);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
        <widget class="QLabel" name="text_out_2">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>30</y>
           <width>341</width>
           <height>351</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(208, 208, 208);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>231</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Motivo</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>10</y>
           <width>91</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Ver motivo</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bt_btl">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>400</y>
           <width>81</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Back to line</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>10</y>
           <width>41</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Fecha:</string>
         </property>
        </widget>
        <widget class="QLabel" name="fecha_out">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>10</y>
           <width>141</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="geometry">
          <rect>
           <x>95</x>
           <y>402</y>
           <width>194</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(214, 214, 214);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_10">
         <property name="geometry">
          <rect>
           <x>292</x>
           <y>404</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <item>
          <property name="text">
           <string>T1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T3</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>384</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>User</string>
         </property>
        </widget>
        <widget class="QLabel" name="st_enviar_2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>385</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>30</y>
           <width>481</width>
           <height>351</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <widget class="QLabel" name="text_out">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>461</width>
            <height>321</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(208, 208, 208);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="bt_like">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>390</y>
           <width>51</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(255, 253, 121, 255));
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Like</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bt_consultar">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>390</y>
           <width>81</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Consultar</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="buscar_fecha">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>60</y>
           <width>174</width>
           <height>36</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>108</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bt_fecha">
         <property name="geometry">
          <rect>
           <x>177</x>
           <y>60</y>
           <width>31</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(75, 98, 226, 255));</string>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Ver</string>
         </property>
        </widget>
        <widget class="QLabel" name="fecha_ver">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>10</y>
           <width>141</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>10</y>
           <width>41</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Fecha:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>10</y>
           <width>41</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Usr:</string>
         </property>
        </widget>
        <widget class="QLabel" name="usr_ver">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>10</y>
           <width>151</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QTableWidget" name="buscar_db">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>11</y>
           <width>691</width>
           <height>381</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <column>
          <property name="text">
           <string>Fecha</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>User</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Turno</string>
          </property>
         </column>
        </widget>
        <widget class="QPushButton" name="bt_consultar_2">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>400</y>
           <width>141</width>
           <height>36</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(151, 111, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>bubscar</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>399</y>
           <width>326</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>398</y>
           <width>201</width>
           <height>36</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="fix">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>58</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
